spark/src/main/java/org/apache/zeppelin/spark/dep/SparkDependencyContext.java

```diff
@@ -64,6 +67,8 @@ public SparkDependencyContext(String localRepoPath, String additionalRemoteRepos
   }
 
   public Dependency load(String lib) {
+    Console.println("DepInterpreter(%dep) deprecated. "
```

Comment:
Why do you use Console here?
Isnt it LOG better? so you can print log like `LOG.warning('deprecated blablalba')`?